_posts/2023-12-11-augmented-reality-in-travel-applications.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: "Our previous experiments with augmented reality (AR) in travel applications"
+date: 2023-12-11 00:00:00 +0200
+categories: javascript motivation product
+author_name: Adrian Schmid
+read_time: 5
+feature_image: posts/2023-12-11-augmented-reality-in-travel-applications/augmented-reality.jpg
+---
+
+Since its inception, HolidayCheck has thrived on user-generated content (UGC), which remains our cornerstone. Our commitment to elevating this UGC experience continually drives us to innovate for our holidaymakers. In this spirit, we've been exploring fresh avenues, such as integrating interactive, geo-based experiences accessible right from the browser. This exploration opens many possibilities, like offering immersive hotel room previews, comprehensive travel guides, and more to enrich the traveler's journey with us.
+
+## AR with geolocalization in browsers
+
+Fast and pragmatic MVP development on smartphones requires a robust and feature-rich toolkit. [Apple's ARKit](https://developer.apple.com/augmented-reality/arkit/) and [Google's ARCore](https://developers.google.com/ar?hl=en) came into question to support native hardware (i.e. the camera in our case) in the best possible way. Nevertheless, I opted for [AR.js](https://github.com/AR-js-org/AR.js), a browser-based solution, due to its simplicity in terms of quick setup and speed of experimentation. At the time, December 2021, the framework was already almost 2 years young and quite mature.
```

Comment:
"was already almost 2 years young and quite mature" - this reads a bit odd, IMO.

How about something like: "was already 2 years in the making, and quite mature".
2 years doesn't have to mean it is young, or immature. On the contrary. Hence the proposed change.